packages/babel-parser/src/parser/expression.js

```diff
@@ -931,8 +933,8 @@ export default class ExpressionParser extends LValParser {
       elts.push(
         this.parseExprListItem(
           false,
-          possibleAsyncArrow ? new ExpressionErrors() : undefined,
-          possibleAsyncArrow ? { start: 0 } : undefined,
+          refExpressionErrors || new ExpressionErrors(),
+          { start: 0 },
```

Comment:
Not related to this PR: The `refNeedsArrowPos` argument here seems to be always disabled. We could consider removing this variable.

Reply:
Do you prefer that we tackle this one in another PR?

Reply:
It can be done in a follow-up PR

Reply:
Should I create an issue ?

Reply:
If you want to just open a new PR (if you still want to work on it), it's ok to just open a PR without an issue!

Reply:
I'll ^^